app/scripts/extract_texts_script.py

```diff
@@ -133,128 +168,30 @@ def get_matched_text(value_string, matched_expressions):
-    # result_tour = list(filter(({}).__ne__, result_tour))
-    # print(len(result_tour))
-    result_tour = [i for i in result_tour if i]
-    result_tour = [i for n, i in enumerate(result_tour) if i not in result_tour[n + 1:]]
-    print(len(result_tour))
-    # ---------------------------------------------------------------------------------------
-
-    # Result from Data list file
-    #print(len(result_data_list))
-    result_data_list = [i for i in result_data_list if i]
-    result_data_list = [i for n, i in enumerate(result_data_list) if i not in result_data_list[n + 1:]]
-    print(len(result_data_list))
-    # ---------------------------------------------------------------------------------------
-
-    with open(path + '//output//output_template.json', 'w', encoding='utf-8') as json_file:
-        json.dump(result_temp, json_file, ensure_ascii=False, indent=2)
-
-    with open(path + '//output//output_global.json', 'w', encoding='utf-8') as json_file:
-        json.dump(result_glob, json_file, ensure_ascii=False, indent=2)
-
-    with open(path + '//output//output_tour.json', 'w', encoding='utf-8') as json_file:
-        json.dump(result_tour, json_file, ensure_ascii=False, indent=2)
-
-    with open(path + '//output//output_data_list.json', 'w', encoding='utf-8') as json_file:
-        json.dump(result_data_list, json_file, ensure_ascii=False, indent=2)
-    # ----------------------------------------------------------------------------------------
-    reslt_temp = [d['text'] for d in result_temp if 'text' in d]
-    print('Number of characters for translation in template.json: ', sum(len(str(i)) for i in reslt_temp))
-    print('Number of words for translation in template.json: ', sum(len(str(i).split()) for i in reslt_temp))
-    print('----------------------------------------------------------')
-    reslt_glob = [d['text'] for d in result_glob if 'text' in d]
-    print('Number of characters for translation in global.json: ', sum(len(str(i)) for i in reslt_glob))
-    print('Number of words for translation in global.json: ', sum(len(str(i).split()) for i in reslt_glob))
-    print('----------------------------------------------------------')
-    reslt_tour = [d['text'] for d in result_tour if 'text' in d]
-    print('Number of characters for translation in tour.json: ', sum(len(str(i)) for i in reslt_tour))
-    print('Number of words for translation in tour.json: ', sum(len(str(i).split()) for i in reslt_tour))
+def remove_empty(results):
+    tmp = [i for i in results if i]
+    print(f'Length after removing empties: {len(results)}')
+    return tmp
+
+def dedupe(results):
+    tmp = [i for n, i in enumerate(results) if i not in results[n + 1:]]
+    print(f'Length after removing duplicates: {len(results)}')
+    return tmp
+
+def save_results(src, results):
+    out_dir = Path(os.getcwd()) / 'output'
+    out_dir.mkdir(parents=True, exist_ok=True)
+    file_name = 'output.json' if src == 'all' else f'output_{src}.json'
```

Comment:
Making sure the names in the console log agree with the created json files
```suggestion
    file_name = f'output_{src}.json'
```